test/org/sagebionetworks/bridge/dynamodb/DynamoUploadSchemaTest.java

```diff
@@ -0,0 +1,64 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import org.junit.Test;
+import org.sagebionetworks.bridge.exceptions.InvalidEntityException;
+import org.sagebionetworks.bridge.json.BridgeObjectMapper;
+import org.sagebionetworks.bridge.models.upload.UploadSchema;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+public class DynamoUploadSchemaTest {
```

Comment:
BTW, these tests are actually in org.sagebionetworks.bridge.models.upload.UploadSchemaTest. If that causes confusion for you, feel free to move those tests here.